package/kedro_viz/integrations/kedro/data_loader.py

```diff
@@ -114,3 +117,17 @@ def load_data(
 
     context = load_context(project_path=project_path, env=env)
     return context.catalog, context.pipelines
+
+def load_all_versions(filepath: str) -> Dict[str, any]:
+    version_list = [x for x in Path(filepath).iterdir() if x.is_dir()]
+    versions = {}
+    for v in version_list:
+        timestamp = datetime.strptime(v.name,VERSION_FORMAT)
+        path = v / Path(filepath).name
+        with open(path) as fs_file:
+            versions.update({timestamp: json.load(fs_file)})
```

Comment:
We're filling an empty dictionary here, so instead of using `update` I'd just do: `versions[timestamp] = json.load(fs_file)`. 

`update` is less efficient and commonly only used when updating a dictionary that already has data in it.